Examples/mod.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Id": "ExampleMod",
+  "DisplayName": "Example Mod",
+  "Author": "the QModManager dev team",
+  "Version": "1.0",
+  "Enable": true,
+  "AssemblyName": "Examples.dll",
+  "EntryMethod": "QModManager.Examples.ExampleMod.Patch"
```

Comment:
While mostly irrelevant, we should consider updating this to the newer format since some modders may want to compile and run it.  
If it's to be used as a guideline, we should keep it up to date.

Reply:
Everything new added to the `mod.json` file is optional.